src/sparseml/transformers/export.py

```diff
@@ -91,12 +91,14 @@
 __all__ = ["export_transformer_to_onnx", "load_task_model"]
 
 MODEL_ONNX_NAME = "model.onnx"
```

Comment:
Introducing the following convention for creating the `deployment` folder:
Files that need to be copied over: `model.onnx`, 'tokenizer_config.json`, 'config.json`. 
Files that can be copied will only raise a warning, not an error when absent: `model.data` (because not every model is >2Gb), `tokenizer.json` because we have seen many times that this file is not mandatory for all the transformer-like models. Confirming this with Tuan.